src/Navigation/WPF/Navigation.WPF/NavigationServiceExtensions.cs

```diff
@@ -235,19 +234,38 @@ public static class NavigationServiceExtensions
         /// <summary>
         /// Removes the registration associated with the given ViewModel interface type
         /// </summary>
-        /// <typeparam name="TViewModelInterface">ViewModel type to remove from registrations</typeparam>
+        /// <typeparam name="TViewModel">ViewModel type to remove from registrations</typeparam>
         /// <param name="navigationService"></param>
         /// <returns>Navigation service instance</returns>
-        public static INavigationService Unregister<TViewModelInterface>(
+        public static INavigationService Unregister<TViewModel>(
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Refactor this method to have parameters matching all the type parameters.](https://app.codacy.com/gh/UltimateCodeMonkeys/CodeMonkeysMVVM/pullRequest?prid=5575746)